src/playground/app.ts

```diff
@@ -0,0 +1,53 @@
+import express from 'express';
+import path from 'path';
+import expressHandlebars from 'express-handlebars';
+
+import dotenv from 'dotenv';
+import fs from 'fs';
+
+if (fs.existsSync('.env')) {
+  console.log('Using .env file to supply config environment variables');
+  dotenv.config({ path: '.env' });
+} else {
+  console.log('Using .env.example file to supply config environment variables');
+  dotenv.config({ path: '.env.example' }); // you can delete this after you create your own .env file!
+}
```

Comment:
https://www.npmjs.com/package/dotenv-cli does this for you. Let's not read from `.env.example` we could copy `.env.example` to `.env` if the latter doesn't exist in the `postinstall` phase

Reply:
Done. 
Note that I needed to create a separate postBuild.ts file so that `npm run playground:build` works properly.